libdeno/internal.h

```diff
@@ -37,6 +39,16 @@ class DenoIsolate {
       isolate_->Dispose();
     }
     delete array_buffer_allocator_;
+    array_buffer_allocator_ = nullptr;
+    isolate_ = nullptr;
+    shared_ = empty_buf;
+    current_args_ = nullptr;
+    snapshot_creator_ = nullptr;
+    global_import_buf_ptr_ = nullptr;
+    pending_promise_events_ = 0;
+    recv_cb_ = nullptr;
+    next_req_id_ = 0;
+    user_data_ = nullptr;
```

Comment:
Is this useful?

Reply:
I don't think this is necessary -- but if we keep it: it's too much code. memset() would be better.

Reply:
I think neither is useful. This is the purpose of ASAN.

That said, if @F001 wants this for debugging, with the expectation that we will remove it later (when ASAN runs on deno) that's okay with me. I prefer this to memset.